mobile/src/main/java/org/openhab/habdroid/background/AlarmChangedReceiver.java

```diff
@@ -0,0 +1,72 @@
+package org.openhab.habdroid.background;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.preference.PreferenceManager;
+import android.util.Log;
+import androidx.work.Constraints;
+import androidx.work.NetworkType;
+import androidx.work.OneTimeWorkRequest;
+import androidx.work.WorkManager;
+
+import org.openhab.habdroid.R;
+
+import static android.content.Context.NOTIFICATION_SERVICE;
+import static org.openhab.habdroid.background.BackgroundUtils.NOTIFICATION_ID_SEND_ALARM_CLOCK;
+import static org.openhab.habdroid.background.BackgroundUtils.NOTIFICATION_TAG_BACKGROUND;
+import static org.openhab.habdroid.background.BackgroundUtils.NOTIFICATION_TAG_BACKGROUND_ERROR;
+import static org.openhab.habdroid.background.BackgroundUtils.WORKER_TAG_SEND_ALARM_CLOCK;
+import static org.openhab.habdroid.background.BackgroundUtils.createNotificationChannels;
+import static org.openhab.habdroid.background.BackgroundUtils.makeBackgroundNotification;
+import static org.openhab.habdroid.util.Constants.PREFERENCE_ALARM_CLOCK_ENABLED;
```

Comment:
I'm not sure, but IMHO we should avoid static imports or at least constrain it to constants, so it's easier to track in the code where stuff is coming from.

Reply:
Yes, I'll remove all static imports for functions.